JudoPayDotNet/JudoPayApi.cs

```diff
@@ -35,13 +36,13 @@ public class JudoPayApi : IJudoPayApi
 
         public JudoPayApi(Func<Type, ILog> logger, IClient client)
         {
-            Payments = new Payments(logger(typeof(Payments)), client);
```

Comment:
so these are the only clients i've set up deDuplication on. seem alright. Didn't think the others needed it/would matter